packages/devtools_app/lib/src/service/resolved_uri_manager.dart

```diff
@@ -0,0 +1,51 @@
+import '../shared/globals.dart';
```

Comment:
add copyright for 2022 at the top of this file. (you can copy and paste from any other file and then just change the year). All files need to have the copyright header at the top.